js/donations-visualiser.js

```diff
@@ -855,20 +855,49 @@ function update(partyNodes, parties, selectedParties, resetControls) {
 
     d3.select("#party_select").selectAll(".checkbox").remove();
 
-    party_checkboxes = d3.select("#party_select").selectAll(".checkbox")
+    parties_grouped = []
+    parties_grouped_search = ["Coalition", "Australian Labor Party", "Australian Greens", "Palmer United Party"]
+    parties.map(function(item){
```

Comment:
bit weird to use map here since you're ignoring the return value. better to use reduce, and then you don't need to define parties_grouped separately.

eg 
```
var parties_grouped = partiesreduce(function(accumulator,` party) {
var index = parties_grouped_search.indexOf(party_map[item].name);
if (index > -1) {
  accumulator[index] = party;
  return accumulator;
}, {});
```